README.md

```diff
@@ -50,6 +50,36 @@ been tested on Cygwin.
 This component license is public domain, a.k.a the  UNLICENSE.
 See details at the sink [location](https://github.com/KjellKod/g3sinks/tree/master/logrotate).
 
+## g3sinks/tracelogging
+sink to allow logging through Windows TraceLogging
+
+### Building and installing g3log_tracelogging
```

Comment:
how about this?
Keep this (or something like it)
```
## g3sinks/tracelogging
Sink to allow logging through Windows TraceLogging. 
For build instructions please see [tracelogging/README](tracelogging/README.md)
```
Remove the rest in this readme file. 

That way you don't have duplicated any info and won't have an issue with it being out of sync between the readmes.

Reply:
Good idea.  I made the change you suggested